framework/include/problems/ReferenceResidualProblem.h

```diff
@@ -43,13 +43,25 @@ class ReferenceResidualProblem : public FEProblem
                             const PetscInt nfuncs,
                             const PetscInt max_funcs,
                             const PetscBool force_iteration,
-                            const Real ref_resid,
+                            const Real initial_residual_before_preset_bcs,
```

Comment:
Was this just poorly named before?

Reply:
Yes.  I changed the name to match the names of parameters in FEProblemBase to make it more clear what it is.  I'm probably the one responsible for the original `ref_resid` name. When I was going through the code, I couldn't remember what that was exactly, so I think this makes it more clear.